src/Perpetuum/Zones/Terrains/Layer.cs

```diff
@@ -142,7 +142,7 @@ public T GetValue(int x, int y)
             return RawData[y*Width + x];
         }
 
-        public void SetValue(int x, int y,T value)
+        public virtual void SetValue(int x, int y,T value)
```

Comment:
I guess this change can be reverted too? We no longer need to override

Reply:
Yes that might have been some local experiments that didn't get cleaned up.